pkg/controller/servicebindingrequest/binder_test.go

```diff
@@ -28,7 +28,8 @@ func TestBinderNew(t *testing.T) {
 	}
 
 	f := mocks.NewFake(t, ns)
-	sbr := f.AddMockedServiceBindingRequest(name, "ref", "", matchLabels)
+	gvr := "apps/v1/deployments" // Group/Version/Resource for sbr
```

Comment:
@pratikjagrut instead of using a hardcoded string, you can programmatically build it, using the following [example](https://github.com/redhat-developer/service-binding-operator/blob/b9616709b8ec3081ecbfbe8ce70ba2076a049b7a/pkg/controller/servicebindingrequest/olm.go#L36):

```go
gvr := olmv1alpha1.SchemeGroupVersion.WithResource(csvResource)
```